tests/teststream.nim

```diff
@@ -1168,7 +1147,9 @@ suite "Stream Transport test suite":
       await sleepAsync(100.milliseconds)
 
       for i in 0 ..< len(futs):
-        if futs[i].failed() and (futs[i].error of TransportUseClosedError):
+        # writes may complete via fast write
+        if futs[i].completed() or (
```

Comment:
This test check become logically wrong, e.g. if it failed or not i do not care... You should fix the test first, you need to perform writes until you get `EAGAIN` and it starts filling queue.

Reply:
Ok, updated the test, though in reality we're lying a little here: it's no big difference whether the data was dropped by chronos or the OS in this case, and yet we're making it seem like so in the test and in the future we returned.